src/index.js

```diff
@@ -53,9 +53,10 @@ var defaultdict = function(){
                 }else{
                     obj[prop] = value;
                 }
+                return true;
```

Comment:
FWIW the documentation doesn't mention this (e.g. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/set) and on my Chrome & node.js this works fine without `return`. Out of interest: what browser or engine do you use?

Reply:
I tried with nodejs version `6.9.1`.

Reply:
Hi @wilzbach 

Acording to this document https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy/handler/set

> The set method should return a boolean value. Return true to indicate that assignment succeeded. If the set method returns false, and the assignment happened in strict-mode code, a TypeError will be thrown.

In my code, i `use strict`, then a TypeError was fired.

Please consider this case!

Thanks.